test/fluree/db/query/fql_test.clj

```diff
@@ -418,6 +419,85 @@
             (is (= [["Marge" 36 :xsd/int]]
                    results))))))))
 
+(deftest ^:integration t-test
+  (testing "querying with t values"
+    (let [conn   (test-utils/create-conn)
+          ledger @(fluree/create conn "people")
+          db1    @(fluree/stage (fluree/db ledger)
+                                {"@context" ["https://ns.flur.ee"
+                                             test-utils/default-context
+                                             {:ex    "http://example.org/ns/"
+                                              :value "@value"
+                                              :type  "@type"}]
+                                 "insert"
+                                 [{:id      :ex/homer
+                                   :ex/name "Homer"
+                                   :ex/age  36}
+                                  {:id      :ex/marge
+                                   :ex/name "Marge"
+                                   :ex/age  {:value 36
+                                             :type  :xsd/int}}
+                                  {:id      :ex/bart
+                                   :ex/name "Bart"
+                                   :ex/age  "forever 10"}]})
+          db1*   @(fluree/commit! ledger db1)
+          db2    @(fluree/stage db1* {"@context" ["https://ns.flur.ee"
+                                                  test-utils/default-context
+                                                  {:ex    "http://example.org/ns/"
+                                                   :value "@value"
+                                                   :type  "@type"}]
```

Comment:
Never thought about aliasing these... I guess we get that mostly for "free" in expansion.